### PR TITLE
Saleor 1775 thumbnail size issues

### PR DESCRIPTION
**Pull request by MarekChoinski**
I want to merge this change because it:

- fixes product gallery preview height
- fixes thumbnail preview height on Order history
- fixes thumbnail height on listing of products

### Screenshots

image

image

image

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1775 Thumbnail size issues](https://app.clickup.com/t/2549495/SALEOR-1775)

File: src/@next/components/molecules/ProductTile/styles.ts [hunk 1]
```diff
@@ -12,6 +12,9 @@ export const Wrapper = styled.div`
   padding: 2.5rem;
   text-align: center;
   max-height: 30rem;
```
Comment (jwm0) [inline src/@next/components/molecules/ProductTile/styles.ts:14]: get rid off `max-height` attr

File: src/@next/components/organisms/ProductGallery/styles.ts [hunk 2]
```diff
@@ -79,6 +79,7 @@ export const Preview = styled.div`
   max-height: 560px;
   overflow: hidden;
   img {
+    max-height: 100%;
```
Comment (jwm0) [inline src/@next/components/organisms/ProductGallery/styles.ts:82]: Is `100%` the max amount or the default? If it's default set it to `height: 100%`

**Comment by krzysztofwolski** [comment]
@MarekChoinski gallery has issue with sizes too https://saleor-1775-thumbnail-size-issues.storefront.saleor.rocks/product/aaaa-sku/498/
Can you fix that as well?

**Comment by MarekChoinski** [comment]
@krzysztofwolski Issue you mentioned is already solved

**Review by dominik-zeglen — APPROVED** [review]

**Review by orzechdev — APPROVED** [review]

**Review by jwm0 — APPROVED** [review]

**Review by mmarkusik — APPROVED** [review]

**Pull request merged**